### PR TITLE
DT-984: May patching

### PR DESCRIPTION
**Pull request by YD-DLUHC**
**Description** 

May patching

### CODE REVIEW

File: auth-service/build.gradle.kts [hunk 1]
```diff
@@ -60,19 +60,19 @@ dependencies {
...
+    implementation("ch.qos.logback:logback-classic:1.5.6")
     implementation("net.logstash.logback:logstash-logback-encoder:7.4") // Structured log encoder
-    implementation("org.jetbrains.kotlinx:kotlinx-coroutines-slf4j:1.8.0")
+    implementation("org.jetbrains.kotlinx:kotlinx-coroutines-slf4j:1.8.1")
 
     // OpenSAML
-    implementation("org.opensaml:opensaml-core:4.3.0")
-    implementation("org.opensaml:opensaml-saml-impl:4.3.0")
+    implementation("org.opensaml:opensaml-core:4.0.1")
+    implementation("org.opensaml:opensaml-saml-impl:4.0.1")
```
Comment (BenRamchandani) [inline auth-service/build.gradle.kts:75]: This is a downgrade! 4.3.2 is the latest. https://mvnrepository.com/artifact/org.opensaml/opensaml-core?repo=shibboleth-releases

**Review by BenRamchandani — CHANGES_REQUESTED** [review]
There are new versions of ktor (2.3.11) and Kotlin (1.9.24) too

File: auth-service/build.gradle.kts (hunk 1 above)
Comment (YD-DLUHC) [reply auth-service/build.gradle.kts]: Upgraded versions

File: auth-service/build.gradle.kts [hunk 2]
```diff
@@ -1,5 +1,5 @@
-val ktorVersion = "2.3.9"
-val kotlinVersion = "1.9.23"
+val ktorVersion = "2.3.11"
+val kotlinVersion = "2.0.0"
```
Comment (BenRamchandani) [inline auth-service/build.gradle.kts:2]: Let's not jump major Kotlin versions yet, can you raise a ticket for it. Make sure the versions just below in the `plugins` block match these

Comment (YD-DLUHC) [reply auth-service/build.gradle.kts]: added versions to block below and ticket raised

File: auth-service/build.gradle.kts [hunk 3]
```diff
@@ -60,19 +60,19 @@ dependencies {
...
+    implementation("ch.qos.logback:logback-classic:1.5.6")
     implementation("net.logstash.logback:logstash-logback-encoder:7.4") // Structured log encoder
-    implementation("org.jetbrains.kotlinx:kotlinx-coroutines-slf4j:1.8.0")
+    implementation("org.jetbrains.kotlinx:kotlinx-coroutines-slf4j:1.8.1")
 
     // OpenSAML
-    implementation("org.opensaml:opensaml-core:4.3.0")
-    implementation("org.opensaml:opensaml-saml-impl:4.3.0")
+    implementation("org.opensaml:opensaml-core:4.3.2")
+    implementation("org.opensaml:opensaml-saml-impl:5.1.2")
```
Comment (BenRamchandani) [inline auth-service/build.gradle.kts:75]: I think we should leave these matching unless you're reasonably confident 4 and 5 will work together
```suggestion
    implementation("org.opensaml:opensaml-core:4.3.2")
    implementation("org.opensaml:opensaml-saml-impl:4.3.2")
```

**Review by BenRamchandani — APPROVED** [review]

**Pull request merged**